src/insetUtilities.js

```diff
@@ -1,21 +1,26 @@
-const nameClass = require('./util/nameClass.js');
-
-module.exports = (theme) => {
-  const generators = [
-    ([modifier, size]) => ({
-      ['[dir="rtl"] ' + nameClass('start', modifier)]: {
-        right: size,
-      },
-      ['[dir="rtl"] ' + nameClass('end', modifier)]: {
-        left: size,
-      },
-      ['[dir="ltr"] ' + nameClass('end', modifier)]: {
-        right: size,
-      },
-      ['[dir="ltr"] ' + nameClass('start', modifier)]: {
-        left: size,
-      },
-    }),
-  ];
-  return generators.flatMap(generator => Object.entries(theme('inset')).flatMap(generator));
+module.exports = ({ matchUtilities, theme }) => {
+  matchUtilities(
+    {
+      start: (value) => ({
+        '[dir="rtl"] &': {
+          right: value,
+        },
+        '[dir="left"] &': {
+          left: value,
+        },
+      }),
+      end: (value) => ({
+        '[dir="rtl"] &': {
+          left: value,
+        },
+        '[dir="left"] &': {
+          right: value,
+        },
```

Comment:
This should be `[dir="ltr"]` as well.

Reply:
Why this suggestion wasn't fixed? It needs to be `ltr` instead of `left`.